client/reader/stream/reader-sidebar-followed-sites/index.jsx

```diff
@@ -111,34 +81,22 @@ export class ReaderSidebarFollowedSites extends Component {
 		}
 
 		return (
-			<ExpandableSidebarMenu
-				expanded={ this.props.isFollowingOpen }
```

Comment:
Seems like with this removal, the `isFollowingOpen` prop below isn't used anymore.

Reply:
Good eye. Thanks for calling this out. Removed in https://github.com/Automattic/wp-calypso/pull/70667/commits/325331f318d1b79d8447f938eecb151aaa70eb68